ush/test/bufr2ioda.sh

```diff
@@ -0,0 +1,206 @@
+#!/bin/bash
```

Comment:
general question, @emilyhcliu , do we want to set up `ctests` so that a user just needs to run `ctest` to test like we do for JEDI repos?

Reply:
Yes.  We will add CTests in obsForge.